vm/interpreter/src/gas_block_store.rs

```diff
@@ -84,9 +84,9 @@ mod tests {
     use super::*;
     use cid::multihash::Code::Blake2b256;
     use forest_db::MemoryDB;
-    use forest_encoding::to_vec;
     use fvm::gas::price_list_by_network_version;
     use fvm::gas::Gas;
```

Comment:
You didn't touch this code but it stood out.

```suggestion
    use fvm::gas::{price_list_by_network_version, Gas};
```